packages/nimble-components/src/table-column/mixins/tests/sortable-column.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { customElement } from '@microsoft/fast-element';
+import {
+    fixture,
+    Fixture,
+    uniqueElementName
+} from '../../../utilities/tests/fixture';
+import { TableColumn } from '../../base';
+import {
+    tableColumnEmptyCellViewTag,
+    tableColumnEmptyGroupHeaderViewTag
+} from '../../base/tests/table-column.fixtures';
+import type { ColumnInternalsOptions } from '../../base/models/column-internals';
+import { ColumnValidator } from '../../base/models/column-validator';
+import { mixinSortableColumnAPI } from '../sortable-column';
+import { TableColumnSortDirection } from '../../../table/types';
+
+const columnName = uniqueElementName();
+@customElement({
+    name: columnName
+})
+class TestTableColumn extends mixinSortableColumnAPI(TableColumn) {
+    protected override getColumnInternalsOptions(): ColumnInternalsOptions {
+        return {
+            cellRecordFieldNames: [],
+            cellViewTag: tableColumnEmptyCellViewTag,
+            groupHeaderViewTag: tableColumnEmptyGroupHeaderViewTag,
+            delegatedEvents: [],
+            validator: new ColumnValidator<[]>([])
+        };
+    }
+}
+
+// prettier-ignore
+async function setup(): Promise<Fixture<TestTableColumn>> {
+    return fixture(columnName);
+}
```

Comment:
no action needed (unless the ignore was accidental): prettier-ignore only impacts the following "node" right, i.e. just this function.. how badly does it mess this function up lol?

Reply:
I had copied this from the other mixin tests, but it doesn't look like it's actually needed. I'll remove it from all of them.